mvg/utils/response_processing.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module contains helper functions for manipulating the response from endpoints.
+"""
+
+from enum import Enum
+from typing import Dict, Union, Callable
+
+
+class SortOrder(Enum):
+    ASC = "asc"
+    DESC = "desc"
+
+
+def get_paginated_items(
+    request: Callable, url: str, params: Dict[str, Union[int, SortOrder]]
```

Comment:
We could use `Dict` for params if we do not want to define a custom type for `params`. If we just use `Dict` as the type, then in the docstring you can describe each key in `params`

Reply:
I will change to `Dict` since we are forwarding the `params` without knowing about its content.